augur/cli/run.py

```diff
@@ -32,13 +36,29 @@ def cli(disable_housekeeper, skip_cleanup):
     else:
         logger.debug("Skipping process cleanup")
 
+    if logstash or logstash_with_cleanup:
+        augur_home = os.getenv('AUGUR_HOME', "")
```

Comment:
There is a package-wide variable called `ROOT_AUGUR_DIRECTORY` that the rest of Augur uses to determine where it's currently located. You can import it with `from augur import ROOT_AUGUR_DIRECTORY`; can you use that instead of this environment variable, since the package variable already exists?